src/lib.ts

```diff
@@ -294,3 +294,24 @@ export function tryEnsureLucky(): boolean {
   }
   return false;
 }
+
+export function isChronoWorthIt(): boolean {
+  const currentAdventures = myAdventures();
+  let futureAdventures = currentAdventures;
+  let currentEnergy = YouRobot.energy();
+  let numEnergyCollects = 0;
+
+  while (futureAdventures > 0) {
+    futureAdventures -= 1;
+    currentEnergy += Math.round(YouRobot.expectedEnergyNextCollect() * 0.85 ** numEnergyCollects);
```

Comment:
expectedEnergyNextCollect already rounds, so this would be more accurate if you don't call it